homeassistant/components/kraken/sensor.py

```diff
@@ -121,6 +121,7 @@ def __init__(
         self._available = True
 
         self._attr_device_info = DeviceInfo(
+            configuration_url="https://www.kraken.com/",
```

Comment:
can this be deeplinked?

Reply:
Theoretically it should be possible. For example `Bitcoin->US Dollar` can be tracked here: 
* [https://trade.kraken.com/markets?quotes=usd&assets=btc](https://trade.kraken.com/markets?quotes=usd&assets=btc) 
*  [https://trade.kraken.com/charts/KRAKEN:BTC-USD](https://trade.kraken.com/charts/KRAKEN:BTC-USD)

However as you can see in the URL parameters it uses `BTC` as a Bitcoin identifier. But the Kraken API which is used for this integration returns `XBT` instead. So Kraken itself uses different identifiers for some coins.

I currently see three options:

1. Merge this PR so we have at least a generic `https://www.kraken.com/`
2. Change the URL to a generic `https://trade.kraken.com/markets` or `https://trade.kraken.com/charts`
3. Close this PR until I find out why Kraken uses different identifiers and how to work around that.

Reply:
I just found out that the Urls under `trade.kraken.com` use `https://api.cryptowat.ch/` under the hood. That explains the different identifiers.

Reply:
I'm fine with this URL as well, I'm just trying to tickle for improvement. Which one would you like to stick with?

Reply:
The one as it currently is. Kraken.com